tests/io/mmap.rs

```diff
@@ -142,3 +142,18 @@ fn test_madvise() {
         munmap(addr, 8192).unwrap();
     }
 }
+
+#[test]
+fn test_msync() {
```

Comment:
Putting the test here is a bit too restrictive as the file is:
```rust
#![cfg(not(any(target_os = "redox", target_os = "wasi")))]
```
but `msync` is supported on `redox`. Not sure if it's a big deal at all.

Reply:
Finished!